packages/candid/src/idl.ts

```diff
@@ -172,10 +172,35 @@ export abstract class Visitor<D, R> {
   }
 }
 
+// we try to use hard to guess names to avoid collisions with other types
```

Comment:
```suggestion
// we try to use hard-to-accidentally-pick names to avoid collisions with other types
```
'guess' makes it sound like a security feature